### PR TITLE
combine bonuses to one page

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

Feature:
- Combine all bonuses to one page

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/components/dashboard/components/RewardUserForm/index.jsx [hunk 1]
```diff
@@ -1,89 +1,64 @@
...
 
   onSubmit = (values, formikBag) => {
-    const { amount } = values
-    const { setBonus } = this.props
-    setBonus(amount)
-    formikBag.resetForm({ amount })
+    const { initialValues } = this.props
+    if (!isEqual(initialValues.joinBonus, values.joinBonus)) {
+      this.props.setJoinBonus(values.joinBonus.amount, values.joinBonus.isActive)
+      formikBag.resetForm(values)
```
Comment (leonprou) [inline dapp/src/components/dashboard/components/RewardUserForm/index.jsx:23]: not sure why do you need to reserForm on each change. Something technical?

**Review by leonprou — COMMENTED** [review]
From here https://github.com/fuseio/fuse-studio/pull/447/files#diff-f795607d4ea67a038b10168f08bd402eR119 looks like you introduced a new plugin `bonuses` for the community, that will replace the old bonus plugins. If that's tru:
- Does it mean we need to migrate are current services that use the old plugins?
- Any changes in the wallet to support this?

File: dapp/src/components/dashboard/components/RewardUserForm/index.jsx (hunk 1 above)
Comment (LiorAgnin) [reply dapp/src/components/dashboard/components/RewardUserForm/index.jsx]: Yeah you're right.
I fixed it.

**Comment by LiorAgnin** [comment]
> From here https://github.com/fuseio/fuse-studio/pull/447/files#diff-f795607d4ea67a038b10168f08bd402eR119 looks like you introduced a new plugin `bonuses` for the community, that will replace the old bonus plugins. If that's tru:
> 
> * Does it mean we need to migrate are current services that use the old plugins?
> * Any changes in the wallet to support this?

No changes need!

**Review by leonprou — APPROVED** [review]
cool, so LGTM 👍

**Pull request merged**